nall/GNUmakefile

```diff
@@ -149,7 +149,9 @@ ifeq ($(symbols),true)
   flags += -g
   ifeq ($(platform),windows)
     ifeq ($(findstring clang++,$(compiler)),clang++)
-      flags   += -gcodeview
+      ifneq ($(symformat),gdb)
```

Comment:
You might want to call this `dwarf` since that's the actual name of the format and the explicit override `-gdwarf`. You might actually want to provide that override here; I think Clang will default to -gcodeview under certain conditions.

Reply:
Updated to specifically use -gdwarf flag when 'symformat=dwarf` is specified, otherwise it defaults to CodeView format (current default).